dataprofiler/tests/profilers/test_profiler_options.py

```diff
@@ -124,7 +124,9 @@ def test_disabling_vocab(self, vocab_mock, *mocks):
         vocab_mock.assert_not_called()
 
         # Check to see default options enable vocab
-        profile = Profiler(self.data)
+        multi_options = ProfilerOptions()
+        multi_options.structured_options.multiprocess.is_enabled = False
```

Comment:
so, this test doesn't work when the multiprocess is enabled?

Reply:
it that's true can we set default value for multiprocess as False?

Reply:
Multiprocess is a Python standard library. I don't see a reason to set it to false as, if you have anything >3 cores at the moment this should be faster.

Regarding why this test is single threaded: I set it to false because as it's a sub process the test cannot evaluate the number of times a function is executed (it doesn't own the process). You have to "own" the process for this library to see how often a given function is executed.